src/shared/common/forms/composite/somu-list.jsx

```diff
@@ -105,6 +105,31 @@ class SomuTableRenderer {
         </>);
     }
 
+    renderFoiAppeal(somuItem) {
```

Comment:
The method doesn't seem to be geared to FOI other than we're the first to use it... can it be chanced easily to just `renderAppeal`?

Reply:
I have addressed this